src/views/RecIM/components/Forms/CreateActivityForm/index.js

```diff
@@ -187,7 +188,7 @@ const CreateActivityForm = ({
       (sport) => sport.Name === activityCreationRequest.sportID,
     )[0].ID;
 
-    createNewActivity(activityCreationRequest).then(() => {
+    createActivity(activityCreationRequest).then(() => {
```

Comment:
This success message should only be shown if the createActivity data post operation was successful. Otherwise, you should show an error message. Please confirm that the value returned from `createActivity()` is checked to confirm creation.

Reply:
This is not necessarily in the scope of my PR as anything to do with the forms should be handled by the PRs implementing the forms. I'll make a note of this an request a fix to someone in the team.